torchvision/models/detection/faster_rcnn.py

```diff
@@ -315,12 +315,17 @@ def fasterrcnn_resnet50_fpn(pretrained=False, progress=True,
         - labels (``Int64Tensor[N]``): the predicted labels for each image
         - scores (``Tensor[N]``): the scores or each prediction
 
+    Faster R-CNN is exportable to ONNX for a batch size of 1 with inputs images of fixed size.
+
     Example::
 
         >>> model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=True)
         >>> model.eval()
         >>> x = [torch.rand(3, 300, 400), torch.rand(3, 500, 400)]
         >>> predictions = model(x)
+        >>>
+        >>> # optionally, if you want to export the model to ONNX:
+        >>> torch.onnx.export(model, x, "faster_rcnn.onnx", opset_version = 11)
```

Comment:
The example is fine, although you mention before that it is exportable with a batch size of 1, but here `x` has a batch size of 2. Not a big deal though

Reply:
you are right. I updated the doc for "fixed batch size" which is more correct (can be 1, or other as long as the new inputs have the same batch size)